lib/location/city.ex

```diff
@@ -1,17 +1,23 @@
 defmodule Location.City do
   @ets_table :geonames
+  @ets_table_by_label :geonames_by_label
 
   defstruct [:id, :name, :country_code]
 
   def load(heir) do
+    :ets.new(@ets_table, [:named_table, :compressed, {:heir, heir, []}])
+    :ets.new(@ets_table_by_label, [:named_table, :compressed, {:heir, heir, []}])
```

Comment:
https://github.com/plausible/analytics/pull/2608 is still open so it's not too late :)

I used `:compressed` because the original ETS table (`:geonames`) already had this option. The file we read from to populate this ETS table has 4.3M in size. Do you think we should remove the `compressed` option?

Also, I believe we need `heir` because this ETS table is created asynchronously from another process (see https://github.com/plausible/location/blob/main/lib/location.ex#L20)

Reply:
Thanks for coming back to me. Here, specifically for google imports, I don't think any of this matters much - I was missing the context. Correct me if I'm wrong but imports are serialized and no concurrent readers use this lookup, right?
 
However, to make this future proof, I think we could get rid of compression, add `read_concurrency` and make the table `ordered_set`.

Reply:
https://github.com/plausible/location/pull/4 :)